webpack.config.js

```diff
@@ -10,4 +11,9 @@ module.exports = {
     path: path.resolve(__dirname, 'dist'),
     library: 'Kontist'
   },
+  plugins: [
+    new CopyPlugin([
+      { from: './lib/graphql/schema.flow.js', to: './lib/graphql/schema.flow.js'}
```

Comment:
our build steps are `tsc && webpack` and typescript compiler can't handle this "flow" typed file, even if I `allowJs: true`. So I think the best approach is simply to copy it as is in the build folder with webpack